app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,47 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.UserData;
+import com.codesoom.assignment.exception.UserNotFoundException;
+import com.codesoom.assignment.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public User getUser(Long id) {
+        return userRepository.findById(id).orElseThrow(() -> new UserNotFoundException(id));
```

Comment:
적절히 개행해서 리뷰어가(동료들이) 코드를 읽기 편하게 해주세요.
어떤 기준으로 개행해야 할 지 잘 모르겠다면 이렇게 해보세요. 
한 줄에 `.`이 한 개씩만 오게 하는 겁니다.

```suggestion
        return userRepository.findById(id)
            .orElseThrow(() -> new UserNotFoundException(id));
```

Reply:
` given(userRepository.findById(eq(givenId))).willReturn(Optional.of(givenUser));`

해당 소스가 고민이였는데요!
저는 계속봐서 그런가 한 줄로 이해하기 쉬운데 약간 긴 느낌이 있는거 같기도 하구요.

Reply:
이런 건 고민하지 말고 그냥 개행하시면 돼요.